### ISSUE TITLE
Logging level configuration of jvm module

### ISSUE THREAD

**Issue by peperc** [fracpete/python-weka-wrapper3#40]
I suggest to implement (if there isn't already implemented) an option to configure the logging level of the jvm module, maybe an argument at the jvm.start(). At default is is set to DEBUG.

```python
import weka.core.jvm as jvm
jvm.start()
```

Prints:

> DEBUG:weka.core.jvm:Adding bundled jars
> DEBUG:weka.core.jvm:Classpath=[...]

Thank you!

**Comment by fracpete** [comment]
Only the logger responsible for `weka.core.jvm` is set to `DEBUG`. That way, it is easier to determine what went wrong when people fire up the JVM. If you want to turn it off, simply do this:

```python
import logging
import weka.core.jvm as jvm

jvm.logger.setLevel(logging.INFO)
jvm.start(packages=True)

# other statements

jvm.stop()
```

However, I will add another parameter to the `start(...)` method, allowing you to set the logging level through there as well.

**Comment by fracpete** [comment]
Implemented with 8dd162f616b6de4ec3784f7e514208e14da5d00f
